src/ctapipe/calib/camera/extractor.py

```diff
@@ -0,0 +1,267 @@
+"""
+Extraction algorithms to compute the statistics from a chunk of images
+"""
+
+__all__ = [
+    "StatisticsExtractor",
+    "PlainExtractor",
+    "SigmaClippingExtractor",
+]
+
+from abc import abstractmethod
+
+import numpy as np
+from astropy.stats import sigma_clipped_stats
+
+from ctapipe.containers import StatisticsContainer
+from ctapipe.core import TelescopeComponent
+from ctapipe.core.traits import (
+    Int,
+    List,
+)
+
+
+class StatisticsExtractor(TelescopeComponent):
+    """Base StatisticsExtractor component"""
+
+    chunk_size = Int(
+        2500,
+        help="Size of the chunk used for the calculation of the statistical values",
+    ).tag(config=True)
+    median_cut_outliers = List(
+        [-0.3, 0.3],
+        help="""Interval of accepted values \\
+                (fraction with respect to camera median value)""",
```

Comment:
From the values and the description here, it is not clear to me how an outlier will be defined.